test/conv.jl

```diff
@@ -36,19 +38,19 @@ using NNlib: DenseConvDims
             gputest((x, w) -> NNlib.conv(x, w, cdims; alpha=2.0), x, w, checkgrad=false) # TODO
             gputest((y, w) -> NNlib.∇conv_data(y, w, cdims; alpha=2.0), y, w, checkgrad=false) # TODO
             gputest((x, y) -> NNlib.∇conv_filter(x, y, cdims; alpha=2.0), x, y, checkgrad=false) # TODO
-            
+
             gputest((y, x, w) -> NNlib.conv!(copy(y), x, w, cdims; beta=2.0), y, x, w, checkgrad=false) # TODO
             # @test_broken gputest((x, y, w) -> NNlib.∇conv_data!(copy(x), y, w, cdims; beta=2.0), x, y, w, checkgrad=false) #TODO
             gputest((w, x, y) -> NNlib.∇conv_filter!(copy(w), x, y, cdims; beta=2.0), w, x, y, checkgrad=false) # TODO
         end
+    end
 
-        # CPU implementation of ∇conv_bias!
-        db = zeros(Float64, 1, 1, 3, 1)
-        dy = randn(Float64, 8, 8, 3, 1)
-        function NNlibCUDA.∇conv_bias!(db, dy)
-            db .= sum(dy, dims=1:(ndims(dy)-2))
-            return db
-        end
-        gputest(NNlibCUDA.∇conv_bias!, db, dy, checkgrad=false)
+    # CPU implementation of ∇conv_bias!
+    db = zeros(Float64, 1, 1, 3, 1)
+    dy = randn(Float64, 8, 8, 3, 1)
+    function NNlibCUDA.∇conv_bias!(db, dy)
+        db .= sum(dy, dims=1:(ndims(dy)-2))
```

Comment:
```suggestion
        db .= sum(dy, dims = 1:(ndims(dy)-2))
```

Am I missing something or is this function not testing anything related to nnlib or conv or bias. Perhaps testing that the GPU kernel exists?

Reply:
Seems so. I just moved it out of the loop, since it does not depend on it.